regparser/commands/tests.py

```diff
@@ -0,0 +1,51 @@
+import click
+import importlib
+import nose
+import os
+import requests_cache
+from stevedore import extension
+from stevedore.exception import NoMatches
+
+
+def get_stevedore_module_names(namespace):
+    names = []
```

Comment:
This function feels like boiler plate rather than being specific to running these tests. Should it exist elsewhere?